### PR TITLE
Update Slovak translation

### PR DESCRIPTION
**Pull request by kubalav**

### CODE REVIEW

File: src/_locales/sk/messages.yml [hunk 1]
```diff
@@ -290,7 +297,7 @@ labelIndentUnit:
   touched: false
 labelInjectionMode:
   description: Label for default option to inject scripts.
-  message: Predvolený režim injektovania
+  message: Predvolený režim vloženia:
```
Comment (gera2ld) [inline src/_locales/sk/messages.yml]: Quote needed.

Comment (kubalav) [reply src/_locales/sk/messages.yml]: Fixed.

File: src/_locales/sk/messages.yml [hunk 2]
```diff
@@ -463,7 +470,7 @@ msgImported:
   description: >-
     Message shown after import. There is an argument referring to the count of
     scripts imported.
-  message: 'Importovaných položiek: $1'
+  message: Importovaných položiek: $1'
```
Comment (gera2ld) [inline src/_locales/sk/messages.yml]: Invalid.
You can see errors in the travis-ci details.

**Comment by kubalav** [comment]
$1 have to be at first position - it is not slavic-friendly :(

**Comment by gera2ld** [comment]
> $1 have to be at first position - it is not slavic-friendly :(

Is there a better way to express this?

**Comment by kubalav** [comment]
Like before - Importovaných položiek $1. If number is at fist place I have to use declension.

In English:
$1 item(s) are imported. = one imported item, many imported items

In Slovak:
$1 importovaná/é/ých položka/ky/iek = 1 importovaná položka, 2-4 importované položky, 5-X importovaných položiek.

You can also write in English:
Imported items: $1
or
Importovaných položiek: $1

**Comment by gera2ld** [comment]
You don't have to match the English version exactly. You can replace the whole message with one that looks good in your language.
That's why I put `$1` in the message to make it a sentence instead of just offering a phrase.

**Comment by kubalav** [comment]
I tried it, but Travis check not passed.
Look [commit #4](https://github.com/violentmonkey/violentmonkey/pull/679/commits/4e4e24d6f3e77270c01595c76e2feec2a666ff41)

**Comment by gera2ld** [comment]
That's because a string with `:` in it needs to be quoted.

**Comment by kubalav** [comment]
You're right. 😀

**Pull request merged**